src/events.rs

```diff
@@ -191,6 +191,32 @@ macro_rules! make_input_event {
     };
 }
 
+macro_rules! make_animation_event {
+    ($name:ident, $type:literal) => {
+        make_event!($name, $type => web_sys::AnimationEvent);
+
+        impl $name {
+            #[inline] pub fn animation_name(&self) -> String { self.event.animation_name() }
+            #[inline] pub fn elapsed_time(&self) -> f32 { self.event.elapsed_time() }
+            #[inline] pub fn pseudo_element(&self) -> String { self.event.pseudo_element() }
+        }
+    };
+}
+
+macro_rules! make_wheel_event {
+    ($name:ident, $type:literal) => {
+        make_mouse_event!($name, $type => web_sys::WheelEvent);
+
+        impl $name {
+            #[inline] pub fn delta_x(&self) -> f64 { self.event.delta_x() }
+            #[inline] pub fn delta_y(&self) -> f64 { self.event.delta_y() }
+            #[inline] pub fn delta_z(&self) -> f64 { self.event.delta_z() }
+            // TODO return enum
+            #[inline] pub fn delta_mode(&self) -> u32 { self.event.delta_mode() }
```

Comment:
Because changing this to an enum is a breaking change, I'd rather leave this out for now.

Reply:
I wanted to leave it up to you to decide, which is why I left the comment. I'm fine either way